src/components/Identifier/IdentifierMasthead/IdentifierMasthead.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { ReactElement } from 'react'
+import classnames from 'classnames'
+
+import { IdentifierLogo } from '../IdentifierLogo/IdentifierLogo'
+
+type Language = 'english' | 'spanish'
+
+interface IdentifierMastheadCopyMap {
+  ariaLabelAgencyIdentifier: string
+  ariaLabelAgencyDescription: string
+  identityDisclaimer: string
+}
+
+const copyMap: Record<Language, IdentifierMastheadCopyMap> = {
+  english: {
+    ariaLabelAgencyIdentifier: 'Agency identifier',
+    ariaLabelAgencyDescription: 'Agency description',
+    identityDisclaimer: 'An official website of the',
+  },
+  spanish: {
+    ariaLabelAgencyIdentifier: 'Identificador de la agencia',
+    ariaLabelAgencyDescription: 'Descripción de la agencia',
+    identityDisclaimer: 'Un sitio web oficial de',
+  },
+}
+
+interface ParentAgency {
+  url: string
+  name: string
+  logo?: string
+}
+
+interface IdentifierMastheadProps {
+  parentAgency: ParentAgency
+  domain: string
+  language?: Language
+  taxpayerDisclaimer?: string
+  className?: string
+}
+
+export const IdentifierMasthead = ({
+  parentAgency,
+  domain,
+  language = 'english',
+  taxpayerDisclaimer,
+  className,
+  ...sectionProps
+}: IdentifierMastheadProps &
+  JSX.IntrinsicElements['section']): React.ReactElement => {
+  const classes = classnames(
+    'usa-identifier__section usa-identifier__section--masthead',
+    className
+  )
+
+  const copy = copyMap[`${language}` as Language]
+
+  return (
+    <section
+      data-testid="identifierMasthead"
+      className={classes}
+      aria-label={copy.ariaLabelAgencyIdentifier}
+      {...sectionProps}>
+      <div className="usa-identifier__container">
+        <div className="usa-identifier__logos">
+          {parentAgency.logo && (
+            <IdentifierLogo
+              agencyUrl={parentAgency.url}
+              agencyName={parentAgency.name}
+              src={parentAgency.logo}
+            />
+          )}
+        </div>
```

Comment:
I know you potentially have a refactor with how this is handled, so I won't expect a change in response to this feedback directly, just throwing it out as a heads up. Feel free to fix alongside your other changes to masthead if that makes more sense to you.

Basically, if there are no logos, the `usa-identifier__logos` `div` should not be present at all. For example, with how the code is written now, that would involve the conditional render of the entire div based on `parentAgency.logo` instead of just the contents of the `div`. 

That appears to be responsible for the extra space when no logos are present:
![image](https://user-images.githubusercontent.com/15805554/112686115-df997680-8e4b-11eb-848c-837ad217c74c.png)